src/components/Callback/Callback.scss

```diff
@@ -0,0 +1,71 @@
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap');
+
+.callback {
+    font-family: 'Ubuntu', sans-serif;
+    height: 100vh;
+    width: 100vw;
+    background-repeat: no-repeat;
+    background-position: center;
+    position: relative;
+    align-items: center;
+
+    &__header {
+        border-bottom: 1px solid #00000014;
```

Comment:
[Q] do we not have any vars css from quill or deriv ui?

Reply:
We do, but I don't think we should set this up here for now because the design is not final yet, this is just a placeholder